app/main/helpers/frameworks.py

```diff
@@ -65,7 +64,7 @@ def get_required_fields(all_fields, answers):
         required_fields.add('SQ3-1k')
 
     # If you answered No to question 26 (established in the UK)
-    if not answers.get('SQ5-2a'):
+    if answers.get('SQ5-2a') is False:
```

Comment:
Yep can do - what is the problem with the `is False` check? The only values that come in are True or False.